### PR TITLE
v3.0,dev/how-to/configure: initialize cluster managed by tidb-operator

### PR DESCRIPTION
**Pull request by tennix**
### What is changed, added or deleted? 

Adding a document about how to initialize TiDB cluster on Kubernetes deployment.

### Which version does your change affect? 

dev and v3.0

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by yikeke** [comment]
**Important Update**: We add a markdownlint static check in the CI check to improve the quality of our documentation, so your unclosed PR might fail the CI check now.

Please click the `details` beside the `ci/circleci: lint — Your tests failed on CircleCI` message, and you can check the details for the failure. After you fix all issues in your PR, the CI check will pass. 

If you have any problem with troubleshooting, please consult @yikeke for help. Thanks for your cooperation~

File: dev/how-to/configure/initialize-cluster.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
```
Comment (lilin90) [inline dev/how-to/configure/initialize-cluster.md]: Please also add this file to `dev/TOC.md`.

File: dev/how-to/configure/initialize-cluster.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+本文介绍如何对 Kubernetes 上的集群进行初始化配置完成初始化账号和密码设置，以及批量自动执行 SQL 语句对数据库进行初始化。
+
+> **注意：**
+>
+> 以下功能只在第一次创建集群时有作用，集群创建之后再设置或修改不会生效。
+
+## 设置初始化账号和密码
+
+集群创建时默认会创建 `root` 账号，但是密码为空，这会带来一些安全性问题。可以通过如下步骤为 `root` 账号设置初始密码
```
Comment (lilin90) [inline dev/how-to/configure/initialize-cluster.md]:
```suggestion
集群创建时默认会创建 `root` 账号，但是密码为空，这会带来一些安全性问题。可以通过如下步骤为 `root` 账号设置初始密码：
```

File: v3.0/how-to/configure/initialize-cluster.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+本文介绍如何对 Kubernetes 上的集群进行初始化配置完成初始化账号和密码设置，以及批量自动执行 SQL 语句对数据库进行初始化。
+
+> **注意：**
+>
+> 以下功能只在第一次创建集群时有作用，集群创建之后再设置或修改不会生效。
+
+## 设置初始化账号和密码
+
+集群创建时默认会创建 `root` 账号，但是密码为空，这会带来一些安全性问题。可以通过如下步骤为 `root` 账号设置初始密码
```
Comment (lilin90) [inline v3.0/how-to/configure/initialize-cluster.md]:
```suggestion
集群创建时默认会创建 `root` 账号，但是密码为空，这会带来一些安全性问题。可以通过如下步骤为 `root` 账号设置初始密码：
```

File: v3.0/TOC.md [hunk 4]
```diff
@@ -40,6 +40,7 @@
   + 配置
     - [时区](how-to/configure/time-zone.md)
     - [内存控制](how-to/configure/memory-control.md)
+	- [初始化集群](how-to/configure/initialize-cluster.md)
```
Comment (lilin90) [inline v3.0/TOC.md]: - Do not use Tab spaces.
- Currently, the website doesn't support this format. If you put "初始化集群" under "内存控制"，the link for "内存控制" cannot be clicked.

**Pull request merged**